markdown/ch7.markdown

```diff
@@ -156,7 +177,7 @@ original state.
 
 Unfortunately, this brings us to the principal difficulty of this kind of
 alignment: sometimes, the MSE has to get worse before it gets better. Have a
-look at the MSE value as the shift gets larger and larger: at around -300
+look at the MSE value as the shift gets larger and larger: at around -400
```

Comment:
Right, I see what you're getting at. I'm going to rewrite that sentence to make it a bit clearer.